docs/repositories/pull-requests.md

```diff
@@ -62,10 +62,24 @@ To access these logs, click the **View logs** link in the [pull request status](
 
 !!! info "This section applies to GitHub repositories only"
 
-If Codacy detects code patterns that have suggested fixes, a **Fix issues** button appears above the issue list. Click this button to commit the suggested fixes to the pull request.
+If Codacy detects code patterns with suggested fixes, a **Fix issues** button appears above the issue list.
 
 ![Fix issues button](images/issues-fix-issues-button.png)
 
+In this case, Codacy generates a patch that enables you to solve all resolvable issues. To apply this patch to the pull request, do the following:
+
+1.  Click the button **Fix issues** to open a modal with a patch that addresses all resolvable issues.
+1.  Copy the patch content to the clipboard.
+1.  Check out the relevant branch in the local repository.
+1.  Apply the patch from the repository's root directory, for example with the `pbpaste | patch` command.
+
+    !!! note
+        These changes are automatically generated. Review them to make sure they're correct.
+
+1.  Push the changes to the repository.
+
+![Fix issues modal](images/issues-fix-issues-modal.png)
```

Comment:
@nicklem Did you place this screenshot here for any particular reason? Would it make sense to place it below the first step (click the button to open the modal)? 🤔